### PR TITLE
fix: add unique indexes on account_changes table

### PR DESCRIPTION
**Pull request by telezhnaya**
Fixes https://github.com/near/near-indexer-for-explorer/issues/74

### CODE REVIEW

**Review by khorolets — APPROVED** [review]
Great finding and nice fix!

> It requires full re-deploy of DB
@telezhnaya Why do you say that?

**Comment by telezhnaya** [comment]
> > It requires full re-deploy of DB

> @telezhnaya Why do you say that?

I could use weird wording, sorry, I meant that we need re-fill it, you know about that. We can't put unique index since the table has duplicates

**Comment by khorolets** [comment]
@telezhnaya ah ok, I thought you have seen something else which will require something more stressful. Just for clarification, could you please add refs to issues you want to address before upgrade? Thanks. Good job!

**Comment by telezhnaya** [comment]
https://github.com/near/near-indexer-for-explorer/issues/111
https://github.com/near/near-indexer-for-explorer/issues/84
https://github.com/near/near-indexer-for-explorer/issues/77

Some of them could not require re-filling, but anyway, it's better to be sure in that
I will ping you directly when it would be the time 🙂

**Review by frol — APPROVED** [review]
Very well done!

**Comment by frol** [comment]
@telezhnaya Can we somehow remove the duplicates without re-indexing the whole history from scratch?

**Comment by telezhnaya** [comment]
@frol I am afraid we will not be happy with that.
There are 105M lines in this table. I am not highload specialist, but it sounds like suffering pain.

Another reason (the main one, actually. It's OK to suffer) is that if I delete something from this table, I will not trust to that data anymore. Each time when I will not find needed lines, I will think that it's not because there is an error; it's because I am stupid and deleted more than I wanted.

To sum up, let's just wait with this migration and re-collect the data when we anyway will have to re-collect it (or when we forget how hard it was during last time)
